library/UnicornHat/unicornhat.py

```diff
@@ -30,6 +30,8 @@
 191,193,194,196,198,200,202,204,206,208,210,212,214,216,218,220,
 222,224,227,229,231,233,235,237,239,241,244,246,248,250,252,255]
 
+colors = {'red':(255,0,0),'green':(0,255,0),'blue':(0,0,255),'yellow':(255,255,0),'magenta':(255,0,255),'cyan':(0,255,255), 'black':(0,0,0),'white':(255,0,0),'gray':(127,127,127),'grey':(127,127,127)}
```

Comment:
`colors` should be uppercase denoting a constant: "COLORS"

For the sake of readability, it's probably okay to format it like so:

```python
COLORS = {
        'red': (255,0,0),
        'green': (0,255,0),
        'blue': (0,0,255),
        'yellow': (255,255,0),
        'magenta': (255,0,255),
        'cyan': (0,255,255),
        'black': (0,0,0),
        'white': (255,0,0),
        'gray': (127,127,127),
        'grey': (127,127,127)
        }
```